helpers/UniversalVerifierContractMigrationHelper.ts

```diff
@@ -20,7 +20,10 @@ export class UniversalVerifierContractMigrationHelper extends ContractMigrationS
     const requestsObj: any[] = [];
 
     if (countRequests > 0) {
-      const requests = await contract.getZKPRequests(0, countRequests);
+      const requests: any = [];
+      for (let i = 0; i < countRequests; i++) {
+        requests.push(await contract.getZKPRequest(i));
+      }
```

Comment:
Can you simplify to only one request? If the one works the other requests almost 100% sure are not broken also. Otherwise this script will be more an more unnecessarily resource consumption as long as requests list grows.
So need to simplify `getDataFromContract`, `checkData` methods and scripts which use them. Shouldn't be hard to fix.